hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyDeletingService.java

```diff
@@ -101,6 +112,21 @@ public BackgroundTaskQueue getTasks() {
     return queue;
   }
 
+  private boolean shouldRun() {
+    if (ozoneManager == null) {
+      // OzoneManager can be null for testing
+      return true;
```

Comment:
By using Java8 method reference, we can remove this, and have a direct check of the method reference instead of calling this method.